src/screens/landing/LandingScreen.tsx

```diff
@@ -10,13 +10,41 @@ export const LandingScreen = () => {
   const i18n = useI18n();
   const navigation = useNavigation();
   const {setLocale} = useStorage();
+
+  const isENFrameworkSupported = async () => {
+    if (Platform.OS === 'ios') {
+      try {
+        await NativeModules.ExposureNotification.isExposureNotificationsFrameworkSupported();
+        return true;
+      } catch (error) {
+        return false;
+      }
+    } else {
+      return true;
+    }
+  };
+
   const toggle = useCallback(
     (newLocale: 'en' | 'fr') => () => {
       setLocale(newLocale);
-      navigation.reset({
-        index: -1,
-        routes: [{name: 'OnboardingNavigator'}],
-      });
+
+      isENFrameworkSupported()
+        .then(isSupported => {
+          if (isSupported) {
+            navigation.reset({
+              index: -1,
+              routes: [{name: 'OnboardingNavigator'}],
+            });
+          } else {
+            Alert.alert(
```

Comment:
1) This will need to go in the locale files - if your not sure how to do that I can help.

2) We'll need to ensure outside of the Alert the user would land on the "something went wrong" homescreen if they make it that far.

Reply:
This commit is just a technical prototype. The way we want to transmit the information through the UI should be something defined by the product team. Do you agree @timarney ?

Reply:
> 2. We'll need to ensure outside of the Alert the user would land on the "something went wrong" homescreen if they make it that far.

This can easily happen if the app was previously installed on a device

Reply:
<img width="368" alt="Screen Shot 2020-12-29 at 4 06 53 PM" src="https://user-images.githubusercontent.com/1763961/103314059-e735b480-49ef-11eb-92bd-0c1247d61fdf.png">
Found a view that was already implemented in the code. It is already translated in both French and English. Will redirect the user to that screen instead of displaying the alert.